packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -1,15 +1,16 @@
-import React from 'react';
+import * as React from 'react';
 import styles from '@patternfly/patternfly/components/ClipboardCopy/clipboard-copy.css';
```

Comment:
Change to `@patternfly/react-styles/css/components/ClipboardCopy/clipboard-copy`